source/sonata_tech.rst

```diff
@@ -83,6 +83,7 @@ Group column represents the HDF group where the dataset is located under /<popul
                                                                                 If the synapse type is excitatory then exc-mini_frequency is used, otherwise inh-mini_frequency is used.
                                                                                 Default to the one provided by the circuit config if not present (Unit is Hz).
     /0                 ``hemisphere``                  utf8       Optional      "left" or "right" hemisphere.
+    /0                 ``placeholder``                 bool       Optional      Whether the node is a placeholder (True) or a detailed (False) one.
```

Comment:
Don't we have a `node_type_id` that we could also use?

Reply:
`node_type_id` is under `/` instead of `/0`, and it would need to be treated in a special way by our tools instead as an additional population property.

Reply:
I agree that it's CSV related, and thus we shouldn't use it.

Instead of a bool, I wonder if we can use the space more efficiently, by encoding multiple pieces of information here; one being whether it's a placeholder, but if the column was `morphology_producer` or something, then we could encode things as `placeholder`, `synthesized`, `biologic`, etc?

Reply:
Yeah, that's a good suggestion. I changed it accordingly.

Reply:
I guess we'll use strings, and make it a `library`?  I'm just trying to think if there is something else that should be folded into it.

Reply:
it could be of interest to have things like bio_axon, synth_axon, etc..

Reply:
by default, synthesis is synthesized dendrites + bio axons/no axons, so synthesis of dendrite + axon is sort of another class, no? Or synthesis of axon + bio dendrites (this never happened so far, but maybe with like purkinje, which are hard to synthesize). But this is just a suggestion from mike's comment above, I'm fine with the basic 3 types

Reply:
@arnaudon , @mgeplf any last words?

Reply:
not from me!